internal/pkg/imports/gen.go

```diff
@@ -14,7 +14,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package imports is used for generating list of imports to optimize use of docker build cache
 package imports
 
+//go:generate bash -c "rm -rf imports_linux.go"
```

Comment:
Do we really need this?

Reply:
Yep, without it we can get such situation:
1. `a.go` imports `foo/bar`
2. we generate imports -> now `imports.go` imports `foo/bar`
3. we delete `a.go` -> so now only `imports.go` imports `foo/bar`
4. we generate imports -> `imports.go` still imports `foo/bar`

Reply:
I think step 3 is incorrect, you need to add step to run go mod tidy.